### ISSUE TITLE
Not perform as expect

### ISSUE THREAD

**Issue by YIHTION** [zhengye1995/Tianchi-2019-Guangdong-Intelligent-identification-of-cloth-defects-rank5#4]
Hi，After I trained the models follow your tips  , I found that this two models not perform very well , even I do the test on a few train set data。I suspect that something is wrong in my training？ Because I noticed that my loss not  drop sharply as I do in other train works。Can you give me some suggestion？ Thanks！
     Best wishes！

**Comment by zhengye1995** [comment]
Did you change the learning rate by mmdetection rules?
Can you share your training log?

This is last lines in my log for 'cascade_rcnn_r50_fpn_70e.py'.

2019-10-15 21:50:15,178 - INFO - Epoch [70][800/1089]	lr: 0.00005, eta: 3:21:21, time: 1.095, data_time: 0.026, memory: 8745, loss_rpn_cls: 0.0048, loss_rpn_bbox: 0.0196, s0.loss_cls: 0.0476, s0.acc: 98.0586, s0.loss_bbox: 0.0466, s1.loss_cls: 0.0219, s1.acc: 98.5019, s1.loss_bbox: 0.0631, s2.loss_cls: 0.0127, s2.acc: 98.3640, s2.loss_bbox: 0.0536, loss: 0.2700
2019-10-15 21:51:09,439 - INFO - Epoch [70][850/1089]	lr: 0.00005, eta: 3:20:27, time: 1.085, data_time: 0.027, memory: 8745, loss_rpn_cls: 0.0010, loss_rpn_bbox: 0.0145, s0.loss_cls: 0.0256, s0.acc: 99.0352, s0.loss_bbox: 0.0262, s1.loss_cls: 0.0108, s1.acc: 99.1487, s1.loss_bbox: 0.0484, s2.loss_cls: 0.0060, s2.acc: 98.9751, s2.loss_bbox: 0.0473, loss: 0.1799
2019-10-15 21:52:03,546 - INFO - Epoch [70][900/1089]	lr: 0.00005, eta: 3:19:33, time: 1.082, data_time: 0.036, memory: 8745, loss_rpn_cls: 0.0081, loss_rpn_bbox: 0.0216, s0.loss_cls: 0.0505, s0.acc: 97.7656, s0.loss_bbox: 0.0568, s1.loss_cls: 0.0236, s1.acc: 98.0001, s1.loss_bbox: 0.0755, s2.loss_cls: 0.0131, s2.acc: 98.0436, s2.loss_bbox: 0.0586, loss: 0.3078
2019-10-15 21:52:59,305 - INFO - Epoch [70][950/1089]	lr: 0.00005, eta: 3:18:43, time: 1.115, data_time: 0.033, memory: 8745, loss_rpn_cls: 0.0019, loss_rpn_bbox: 0.0151, s0.loss_cls: 0.0246, s0.acc: 98.9844, s0.loss_bbox: 0.0263, s1.loss_cls: 0.0097, s1.acc: 99.1840, s1.loss_bbox: 0.0419, s2.loss_cls: 0.0045, s2.acc: 99.2221, s2.loss_bbox: 0.0391, loss: 0.1630
2019-10-15 21:53:51,786 - INFO - Epoch [70][1000/1089]	lr: 0.00005, eta: 3:17:45, time: 1.050, data_time: 0.034, memory: 8745, loss_rpn_cls: 0.0042, loss_rpn_bbox: 0.0122, s0.loss_cls: 0.0425, s0.acc: 98.2109, s0.loss_bbox: 0.0471, s1.loss_cls: 0.0194, s1.acc: 98.4570, s1.loss_bbox: 0.0664, s2.loss_cls: 0.0122, s2.acc: 98.0126, s2.loss_bbox: 0.0512, loss: 0.2552
2019-10-15 21:54:47,230 - INFO - Epoch [70][1050/1089]	lr: 0.00005, eta: 3:16:54, time: 1.109, data_time: 0.074, memory: 8745, loss_rpn_cls: 0.0047, loss_rpn_bbox: 0.0312, s0.loss_cls: 0.0540, s0.acc: 97.7930, s0.loss_bbox: 0.0594, s1.loss_cls: 0.0236, s1.acc: 98.2428, s1.loss_bbox: 0.0781, s2.loss_cls: 0.0115, s2.acc: 98.2743, s2.loss_bbox: 0.0654, loss: 0.3280

**Comment by YIHTION** [comment]
Sorry for late reply。
And this is the last lines in my log for 'cascade_rcnn_r50_fpn_70e.py'。
It doesn't seem to be much different from your log。
I see your train data size is 1089, Is your train_set all the round2 data?
And I did not change the config file from your git。
Or where else（like test module）might I have some problems？
Best wishes for you！

2019-12-11 07:29:37,290 - INFO - Epoch [70][4000/4354]	lr: 0.00005, eta: 0:03:02, time: 0.551, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0033, loss_rpn_bbox: 0.0165, s0.loss_cls: 0.0372, s0.acc: 98.5547, s0.loss_bbox: 0.0390, s1.loss_cls: 0.0174, s1.acc: 98.7564, s1.loss_bbox: 0.0501, s2.loss_cls: 0.0090, s2.acc: 98.5859, s2.loss_bbox: 0.0426, loss: 0.2151
2019-12-11 07:30:05,567 - INFO - Epoch [70][4050/4354]	lr: 0.00005, eta: 0:02:36, time: 0.566, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0092, loss_rpn_bbox: 0.0336, s0.loss_cls: 0.0627, s0.acc: 97.6523, s0.loss_bbox: 0.0586, s1.loss_cls: 0.0303, s1.acc: 97.7595, s1.loss_bbox: 0.0768, s2.loss_cls: 0.0169, s2.acc: 97.3833, s2.loss_bbox: 0.0601, loss: 0.3482
2019-12-11 07:30:34,541 - INFO - Epoch [70][4100/4354]	lr: 0.00005, eta: 0:02:10, time: 0.579, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0070, loss_rpn_bbox: 0.0273, s0.loss_cls: 0.0556, s0.acc: 97.8242, s0.loss_bbox: 0.0520, s1.loss_cls: 0.0271, s1.acc: 97.9925, s1.loss_bbox: 0.0797, s2.loss_cls: 0.0150, s2.acc: 97.9949, s2.loss_bbox: 0.0692, loss: 0.3328
2019-12-11 07:31:02,397 - INFO - Epoch [70][4150/4354]	lr: 0.00005, eta: 0:01:45, time: 0.557, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0066, loss_rpn_bbox: 0.0188, s0.loss_cls: 0.0578, s0.acc: 97.5703, s0.loss_bbox: 0.0645, s1.loss_cls: 0.0278, s1.acc: 97.7466, s1.loss_bbox: 0.0810, s2.loss_cls: 0.0155, s2.acc: 97.5557, s2.loss_bbox: 0.0600, loss: 0.3320
2019-12-11 07:31:33,005 - INFO - Epoch [70][4200/4354]	lr: 0.00005, eta: 0:01:19, time: 0.612, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0105, loss_rpn_bbox: 0.0343, s0.loss_cls: 0.0733, s0.acc: 96.9688, s0.loss_bbox: 0.0776, s1.loss_cls: 0.0347, s1.acc: 97.1353, s1.loss_bbox: 0.0961, s2.loss_cls: 0.0180, s2.acc: 97.0561, s2.loss_bbox: 0.0750, loss: 0.4195
2019-12-11 07:32:01,638 - INFO - Epoch [70][4250/4354]	lr: 0.00005, eta: 0:00:53, time: 0.573, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0055, loss_rpn_bbox: 0.0262, s0.loss_cls: 0.0523, s0.acc: 97.8281, s0.loss_bbox: 0.0613, s1.loss_cls: 0.0230, s1.acc: 98.1552, s1.loss_bbox: 0.0829, s2.loss_cls: 0.0122, s2.acc: 98.1153, s2.loss_bbox: 0.0725, loss: 0.3359
2019-12-11 07:32:30,886 - INFO - Epoch [70][4300/4354]	lr: 0.00005, eta: 0:00:27, time: 0.585, data_time: 0.009, memory: 8913, loss_rpn_cls: 0.0055, loss_rpn_bbox: 0.0283, s0.loss_cls: 0.0650, s0.acc: 97.1211, s0.loss_bbox: 0.0769, s1.loss_cls: 0.0319, s1.acc: 97.3958, s1.loss_bbox: 0.0918, s2.loss_cls: 0.0172, s2.acc: 97.3643, s2.loss_bbox: 0.0714, loss: 0.3880
2019-12-11 07:32:59,202 - INFO - Epoch [70][4350/4354]	lr: 0.00005, eta: 0:00:02, time: 0.566, data_time: 0.008, memory: 8913, loss_rpn_cls: 0.0046, loss_rpn_bbox: 0.0279, s0.loss_cls: 0.0486, s0.acc: 98.1055, s0.loss_bbox: 0.0507, s1.loss_cls: 0.0243, s1.acc: 98.0506, s1.loss_bbox: 0.0656, s2.loss_cls: 0.0131, s2.acc: 97.7996, s2.loss_bbox: 0.0552, loss: 0.2900

**Comment by zhengye1995** [comment]
Sorry for late reply。
Yes, my training data is all round2 data without normal data.
I has 4 2080ti so the dataset size looks like 1089, and your size is 4354, so I guess you used 1 gpu.
So I think you should changle your Lr to lr/4 base your GPU number, my learning rate is lr=0.005,so you should use 0.005/4 = 0.00125, good luck!

**Comment by YIHTION** [comment]
Thank you! I will try it。
